.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -30,12 +30,11 @@ What should happen?
 What happened.
 
 ### Environment
-* Gatsby version (`npm list gatsby`):
-* gatsby-cli version (`gatsby --version`):
-* Node.js version:
-* Operating System:
+
+<!-- Required. Run `gatsby info --clipboard` in your gatsby project directory and paste its contents here. -->
```

Comment:
Maybe mention to update global `gatsby-cli` if command doesn't exist?

Reply:
Great point, but brings a bit of a catch 22 - for anyone to include this info, they have to be on the latest gatsby-cli, whether or not that's where the bug is or not. I know some projects like React Native prefer it this way. Might we include something like an npx command for the latest?

Reply:
I think adding a note saying this command was added in a recent version of gatsby-cli and they might need to upgrade to the latest to get it should be fine.

Also given the rapid pace of new people trying Gatsby, it won't be long before the majority of people using Gatsby have this.

Reply:
Done.